### ISSUE TITLE
SAC policy loss

### ISSUE THREAD

**Issue by rmrafailov** [vitchyr/rlkit#73]
Based on the latest SAC code release the policy loss is given by:

 policy_loss = (alpha*log_pi - q_new_actions).mean()

Which makes sense as empirical estimation of the KL projection loss, besides the alpha parameter. Why is that included? I thought this step, just tries to project the policy into the exp(Q) distribution, so why the alpha?

Thank you.

**Comment by vitchyr** [comment]
See Section 5 of https://arxiv.org/pdf/1812.05905.pdf, which is the updated version of SAC referred to in the README. Basically, this converts "entropy regularized RL" into "entropy constrained RL" where the entropy term is weighted by alpha, an automatically tuned parameter that ensure the entropy is above some threshold.
